#75 removes classname from input and fixes accessibility score on lighthouse

stu562 opened the pull request:
Also adds  aria-label property

stu562 committed:
removes classname from input and fixes accessibility score on lighthouse

abregorivas commented on _includes/forms/contact-us.html:
```diff
@@ -4,20 +4,21 @@ <h3> Contact Us</h3>
   <a href="http://hackforla-slack.herokuapp.com/" rel="noopener" target="_blank">check us out on Slack</a>? Looking to volunteer? </p>
 <form action="thank-you" method="post" class="contact-form" name="Contact Form">
-  <label for="message" class="sr-only">
+  <label for="message">
```
We do not want to remove this class. The text should be hidden but readable for screen readers.

abregorivas commented on _includes/forms/contact-us.html:
```diff
@@ -4,20 +4,21 @@ <h3> Contact Us</h3>
   <div class="form-controls">
     <div class="form-input">
-      <label for="contact-email" class="sr-only">
+      <label for="contact-email">
```
Same as above keep class "sr-only"

abregorivas commented on _includes/forms/hero-signup.html:
```diff
@@ -1,7 +1,12 @@
   <div class="form-input">
-    <label for="hero-signup" class="sr-only"> Enter your email address</label>
-    <input id="hero-signup" type="email" name="email" placeholder="Enter your email address" required />
+    <label for="hero-signup">Enter your email address</label>
```
keep "sr-only" class

abregorivas reviewed (changes requested):
Please add back the "sr-only" classes as they are needed to keep the look of the site the same but still accessible.

stu562 committed:
removed label from contact and hero

stu562 commented:
updated pull request, 
removed label from code. class="sr-only" purpose is to make it not visible. 
the arial label is a text label that is not visible to the screen. Passes audit score and verifying not visible on local enviornment

thekaveman commented:
hey @stu562, I think @abregorivas has it right - we should prefer `sr-only` hidden labels vs. `aria-label`. I've only done a small bit of reading, but it seems that ARIA properties aren't actually *required* to be supported in Screen Readers.

Here's an interesting look at this exact issue across a range of Screen Reader + Browser combinations: https://www.powermapper.com/tests/screen-readers/labelling/input-text-aria-label/

The conclusion is:

> A more reliable alternative is wrapping the INPUT with a LABEL element.

ExperimentsInHonesty commented:
I have reassigned the issue to @abregorivas, so that he can either test the code, or discuss in person on next hack night with @stu562 Steven.

abregorivas commented:
Please remove the label tags for the following 
- contact-us form remove(contact-email, message) label
- hero-signup form remove(hero-signup) label
Add the text that was in labels to an "aria-label" to its corresponding input field.

stu562 committed:
updates contact-us, removes label from textarea, adds aria label to input

ajb176 commented:
You must be a member of the HFLA website team in order to create pull requests. Please see our page on how to join us as a member at HFLA: https://www.hackforla.org/getting-started.  If you have been though onboarding, and feel this message was sent in error, please message us in the #hfla-site team Slack channel with the link to this PR.
